datasets/farsi_news/README.md

```diff
@@ -0,0 +1,20 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- fa
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
+---
```

Comment:
do you mind adding the rest of the dataset card ?

you can leave each paragraph with `[More Information Needed]`